sparse/model.py

```
@@ -0,0 +1,1419 @@
+
+        # === Branch 2: Selected Attention ===
+        qkv_slc = self.qkv_slc(x)
+        q_slc, k_slc, v_slc = (
+            self._split_qkv(
+                qkv_slc, B, N
+            )
+        )
+
+        # Use compressed attention scores for block selection
+        # For simplicity, we use the same compressed K for selection scoring
+        block_indices, _ = (
```

Ruff (0.14.8)

Unpacked variable `block_indices` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)